cypress/support/commands.js

```diff
@@ -48,7 +48,18 @@ Cypress.Commands.add('createRoom', () => {
         method: 'POST',
         url: '/api/room/create'
     }).as('createRoom');
-    cy.get('[data-cy="create-room"]').click();
+    cy.get('[data-cy="Drop-Down"]').select('3');
+    cy.get('[data-cy="create-room"]').click()
+    cy.wait('@createRoom');
+});
+
+Cypress.Commands.add('twoRounds', () => {
```

Comment:
Instead of having this as a separate command, you can use the original create room one but pass in the number of rounds you want, see the `login` command